rust/src/ffi/hashing.rs

```diff
@@ -32,15 +32,20 @@ pub const SC_MD5_HEX_LEN: usize = 32;
 // with .0.
 pub struct SCSha256(Sha256);
 
+extern "C" {
+    #[no_mangle]
+    static g_hash_byte_limit: u32;
```

Comment:
is this essentially an `extern uint32_t g_hash_byte_limit` referring to the C variable? Seems newer Rust doesn't like it, see the Fedora CI failures